ext/project_data.yaml

```diff
@@ -33,21 +33,3 @@ gem_rdoc_options:
   - --main
   - README.md
   - --line-numbers
-gem_platform_dependencies:
-  universal-darwin:
-    gem_runtime_dependencies:
-      CFPropertyList: ['>= 3.0.6', '< 4']
-  x86-mingw32:
-    gem_runtime_dependencies:
-      ffi: '1.15.5'
-      minitar: '~> 0.9'
-  x64-mingw32:
-    gem_runtime_dependencies:
-      ffi: '1.15.5'
-      minitar: '~> 0.9'
-bundle_platforms:
-  universal-darwin: all
-  x86-mingw32: mingw
-  x64-mingw32: x64_mingw
-pre_tasks:
-  'package:apple': 'cfpropertylist'
```

Comment:
This may be used when building the `puppet-agent*dmg` package.

Reply:
Okay I'll add those lines back

Reply:
Sorry I meant lines 36-51 should be deleted (since the gemspec is the source of truth for that information). But lines 52-53 may still be needed? It used to be that if the `package:apple` rake task was called (as defined in the packaging repo), then it would pull in the additional cfpropertylist dependency, see https://github.com/puppetlabs/packaging/blob/1c1adbb5f290a8cdd97c4e2f394534bea30b544c/README.md?plain=1#L615-L619 I don't know if that's actually used any more.

Reply:
Oh got it, that makes sense. I'll put it back just to be safe, I'll also put back the gem_rdoc_options too since that seems like it may be needed for the rdoc rake task and isn't metadata.